### PR TITLE
fix() partitionID strategy fix

### PR DESCRIPTION
**Pull request by jsam**

### CODE REVIEW

File: zbc/client.go [hunk 1]
```diff
@@ -29,6 +31,21 @@ type Client struct {
...
+	leaders, ok := c.Cluster.TopicLeaders[topic]
+	if !ok {
+		c.Topology()
+		leaders, ok = c.Cluster.TopicLeaders[topic]
+		if !ok {
+			return 0, errTopicLeaderNotFound
+		}
+	}
+
+	rnd := rand.New(rand.NewSource(time.Now().UnixNano()))
```
Comment (menski) [inline zbc/client.go]: Noob go question: Is it normal create a new random source on every function call? The client could have a reusable random source?

File: zbc/client.go [hunk 2]
```diff
@@ -191,6 +228,11 @@ func (c *Client) TaskConsumer(topic, lockOwner, taskType string) (chan *Message,
 
 // TopicConsumer opens a subscription on topic and returns a channel where all the SubscribedEvents will arrive.
 func (c *Client) TopicConsumer(topic, subName string) (chan *Message, error) {
+	partitionID, err := c.partitionID(topic)
```
Comment (menski) [inline zbc/client.go]: I assume you don't care at the moment that now the topic/task consumer will create a subscription on a random partition? You asking not judging.

Comment (jsam) [reply zbc/client.go]: Yes. I know.

https://github.com/zeebe-io/zbc-go/issues/48

File: zbc/client.go (hunk 1 above)
Comment (jsam) [reply zbc/client.go]: The default number generator is deterministic, so it'll produce the same sequence of numbers each time by default. To produce varying sequences, give it a seed that changes.

This is temporary until I focus on this https://github.com/zeebe-io/zbc-go/issues/40

**Pull request merged**